panel/src/components/Forms/Input/NumberInput.vue

```diff
@@ -80,21 +80,13 @@ export default {
       return this.step.toString().split(".")[1].length || 0;
     },
     format(value) {
-      // don't convert empty values or the beginning of
```

Comment:
what happens to those string now that start with a minus sign or a zero? od't we have to treat them in a different way? or is that solved too with the new implementaition?

Reply:
They were only treated in the method so far, because the method was executed on input and that would stop them from being entered entirely without this exception. Now the transformation takes place on change and it's no longer a problem to sanitize them with parseInt or parseFloat.